### PR TITLE
Blog: Bioconductor(Outreachy) internship experience

### PR DESCRIPTION
**Pull request by BerylKanali**
@mblue9 @jwokaty 

Here is the blog Kirabo and I have been working on.

### CODE REVIEW

**Comment by mblue9** [comment]
Great! Thank you both. I'll take a look and get back to you later this week.

**Comment by jwokaty** [comment]
@kakopo @BerylKanali This is great. Thanks for promoting Outreachy. It'd be great if we could also promote the next internship round with deadlines for applicants and mentors:

* For interns, application submission deadline: February 6, 2023
* For mentors, project submission deadline: Feb. 24, 2023

Make sure if you're using their content, that you quote it.

I think it would also be nice to know where you're from, your career stage--e.g., just graduated from college or will start graduate school--and how this experience fits in with your future goals/plans. I would also suggest promoting your projects with links so people can see your work. Or if it's like Sweave2Rmd, you could also tell them how to get involved.

**Comment by mblue9** [comment]
Hi @kakopo and @BerylKanali I had a chance to read through it now and agree with Jen, it's great! Also agree with her points, including it would be nice to know a bit about you, you could even include your interests e.g. like "Who I am" [here](https://galaxyproject.org/blog/2022-06-22-outreachy-intro/). If you had a photo to include would be nice so people can see you, or a link to where you have one, but if you don't that's ok.

**Comment by BerylKanali** [comment]
@jwokaty @mblue9 We have made the updates, please take a look and let us know if they are fine.

File: posts/2023-01-22-OutreachyInternship/index.qmd
Comment (jwokaty) [inline posts/2023-01-22-OutreachyInternship/index.qmd:35]: Move lines 31-35 to the end.

File: posts/2023-01-22-OutreachyInternship/index.qmd
Comment (jwokaty) [inline posts/2023-01-22-OutreachyInternship/index.qmd:44]: Here, I think we can mention the projects (lines 58-74) so move that section here.

**Review by jwokaty — CHANGES_REQUESTED** [review]
@BerylKanali @kakopo I made some comments to move some content. I also want to add that I think it's better to put your introductions with your experience rather than separating them into a why we chose bioconductor section. Please consolidate so that we have Kirabo's experience and Beryl's experience.

It's great you included links to your blogs because I didn't think of that!

Please, please include links to the projects you're working on and make sure project and function names they are spelled correctly. I think I saw a misspelling, but I am going to ask you to both proofread your work.

Please also wrap the long text. Check the spelling and address any formatting issues. Let's polish this up for Friday so that we can show it at the core team meeting.

**Comment by BerylKanali** [comment]
@jwokaty We have made the requested changes.

**Comment by mblue9** [comment]
It's looking good to me. I just have a few minor suggestions

- would be nice to say "why" the project is needed e.g. 
    - for sweave2rmd why are the vignettes are being converted? is it to modernise/make easier to maintain/look nicer/be consistent with the majority of packages?
    - for BSGenome why would someone need to forge a BSGenome package? is it if no package exists for a species so they need to make one? is it also to make a version of an existing package with newer annotation?
- I think not everyone reading may be familiar with the term "forge a package" so could you add what that means - it's to make a new one?
- for function names in markdown, it's common to enclose in backticks to highlight in the doc so you could do that for `forgeBSgenomeDataPkg`
- instead of "hundreds and thousands of software packages" you could put >2,000
- for the Conclusion would be nice to add a direct link to the applications, e.g. where you write "apply", to make easier for anyone interested

**Comment by BerylKanali** [comment]
@jwokaty @mblue9 We have updated the blog. Please take a look at the latest changes.

**Comment by jwokaty** [comment]
@kakopo @BerylKanali Did you want to add anything else after today's discussion of the blog? I am going to ask who should approve the PR on the social media channel of the community slack.

**Comment by kozo2** [comment]
Hi @jwokaty @kakopo @BerylKanali 
Thanks for your contribution.
I can approve and merge this PR.
Let me know if you guys are ready for the merge.

**Comment by BerylKanali** [comment]
Hi @kozo2 @jwokaty 
We are currently working on some few edits. The PR will be ready in an 1-2 hours. I will comment when it's ready.
Thanks.

**Comment by kozo2** [comment]
Hi @BerylKanali 
Thanks for the comment.
I understand. I will merge as soon as you are ready.

**Comment by BerylKanali** [comment]
@kozo2 @jwokaty 
The blog  is ready to be merged.

**Comment by kozo2** [comment]
@BerylKanali Thank you!
Now I merged this PR to `main`!

**Comment by kozo2** [comment]
@BerylKanali @kakopo @jwokaty 
Thank you for your contribution!
Now you can check your contribution in https://bioconductor.github.io/biocblog/ !

**Comment by mblue9** [comment]
I'll tweet from Bioconductor about this tomorrow

**Pull request merged**
